docs/api.md

```diff
@@ -372,6 +373,37 @@ Failing requests may be due to the following errors:
 * status code 401, errno 111:  invalid authentication timestamp
 * status code 401, errno 115:  invalid authentication nonce
 
+## GET /v1/account/profile
+
+:lock: OAuth Bearer token with scope `profile`, or HAWK-authenticated with sessionToken
+
+Get the email and locale of a user.
+
+### Request
+
+___Headers___
+
+The request must include an OAuth Bearer token with the scope of `profile`, or a HAWK header that authenticates the request using a `sessionToken` received from `/account/login`.
```

Comment:
This is not entirely accurate; if it only has e.g. `profile:email` scope then the response will only include the `email` field.